.travis.yml

```diff
@@ -4,11 +4,10 @@ python:
   - "2.7"
   - "3.6"
   - "3.7"
-  - "3.8-dev"
+  - "3.8"
 
 matrix:
     allow_failures:
-        - python: "3.8-dev"
```

Comment:
I think removing the entire YAML block would be clearer. This seems like something is missing

Reply:
I was just going to comment about this.

Should I also squash commits?